APITaxi2/schemas.py

```diff
@@ -422,6 +428,18 @@ def check_password(self, password):
         if password:
             return validate.Length(min=8)(password)
 
+    @validates('hail_endpoint_production')
+    def check_endpoint(self, endpoint):
+        """Reject internal and private addresses"""
+        if endpoint:
+            url = urllib.parse.urlparse(endpoint)
+            if not url.netloc or url.scheme not in ('http', 'https'):
+                raise ValidationError("This endpoint is invalid.")
+            # Resolve name to IP
+            ip = ipaddress.ip_address(socket.gethostbyname(url.hostname))
+            if str(ip) in current_app.config.get('APITAXI_IPS') or ip.is_private:
```

Comment:
Do not check APITAXI_IPS. It is not future proof, and it is useless anyway: we do not mind if it is possible to perform a query on our public IPs.

Reply:
C'est cassé de toute façon, en empêchant de mettre localhost, on se bloque pour développer en local, il faudra que je t'en parle.

Reply:
Tu peux utiliser `from flask import current_app ; current_app.debug` qui est à True en local uniquement